tools/test_yaml_valid

```diff
@@ -16,7 +18,14 @@ if (my $err = $@) {
 
 pass("Nothing to do") unless @ARGV;
 for my $file (@ARGV) {
-    my $data = eval { YAML::PP::LoadFile($file) };
+    my $data = eval {
+        my $root_project_dir = dirname(__FILE__) . '/../';
+
+        my $include = YAML::PP::Schema::Include->new(paths => ($root_project_dir));
+        my $ypp     = YAML::PP->new(schema => ['Core', $include, 'Merge']);
+        $include->yp($ypp);
```

Comment:
I think I would do the initializing of the `$ypp` object outside of the loop.
We only need the `load_file` inside the loop.

Reply:
Done.